docs/pages/preview/api-routes.mdx

```diff
@@ -363,3 +363,15 @@ You can now visit your website at the URL provided by Netlify CLI. Running `netl
 If you're using any environment variables or **.env** files, add them to Netlify. You can do this by going to the **Site settings** and adding them to the **Build & deploy** section.
 
 </Step>
+
+## Known Limitations
+
+There are a number of known features that are not currently supported in the API Routes beta release.
+
+### No dynamic imports
+
+API Routes currently work by bundling all code (minus the Node.js built-ins) into a single file. This means that you cannot use any external dependencies that are not bundled with the server. For example, a library like `sharp` which includes multiple platform binaries cannot be used. This will be addressed in a future version.
```

Comment:
```suggestion
API Routes currently work by bundling all code (minus the Node.js built-ins) into a single file. This means that you cannot use any external dependencies that are not bundled with the server. For example, a library such as `sharp`, which includes multiple platform binaries, cannot be used. This will be addressed in a future version.
```

Reply:
Some suggestions on wording as per our writing style guide for concistency.